packages/babel-plugin/src/types.ts

```diff
@@ -116,11 +116,11 @@ export interface State extends PluginPass {
    * Means the `styled` api was found as `styledFunction` - as well as CSS prop is enabled in this module.
    */
   compiledImports?: {
-    ClassNames?: string;
-    css?: string;
-    keyframes?: string;
-    styled?: string;
-    cssMap?: string;
+    ClassNames?: string[];
+    css?: string[];
+    keyframes?: string[];
+    styled?: string[];
+    cssMap?: string[];
   };
 
   usesXcss?: boolean;
```

Comment:
There is also `importedCompiledImports` — @pancaspe87 you're last to touch that line of code... what does it do? It doesn't have a jsdoc description. Should we give it the same treatment or it's something else?

Reply:
don't remember, give me some time to dig into it

Reply:
Answer is possibly [here](https://github.com/atlassian-labs/compiled/pull/1321/files#diff-5b63ee465fe228fb576db7b039cc976de17575ad2fe33c5172fb1e2239818aa1R7)

Reply:
In terms of giving the same treatment, I am not really sure. I think I introduced the change to address [this issue](https://github.com/atlassian-labs/compiled/issues/1316) - which seems specific to imported mixins

Reply:
So if the imported mixins can also use APIs from another module, then... probably should be given the same treatment 😄.

I'll fix it up and add context around the internal state.

Reply:
After digging through the code it doesn't actually check for a specific import specifier. I'll leave it for now.